scripts/build-3rd-parties-dependencies.sh

```diff
@@ -2,22 +2,24 @@
 
 # Usage: run this script from the root of Simplified-iOS repo.
 #
-#     build-3rd-parties-dependencies.sh
+#     ./scripts/build-3rd-parties-dependencies.sh
 #
 # Description: This scripts wipes your Carthage folder, checks out and rebuilds
 #              all Carthage dependencies. It also rebuilds OpenSSL and cURL
 #              from scratch.
 
+echo "Building 3rd party dependencies..."
+
 # update dependencies from Certificates repo
 ./update-certificates.sh
```

Comment:
Shouldn't this line has the `/scripts/` as well?

Reply:
whoops! you're right, not sure how I missed this.

Reply:
fixed both